cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -959,6 +969,18 @@ public Secret generateBrokersSecret() {
         return portList;
     }
 
+    public int getClientPort() {
+        return this.CLIENT_PORT;
+    }
+
+    public int getClientTlsPort() {
+        return this.CLIENT_TLS_PORT;
+    }
+
+    public int getExternalPort() {
+        return this.EXTERNAL_PORT;
```

Comment:
I do not think this works. The external port from the user perspective will be different:
* 443 for Routes
* 9094 for laod balancers
* and 3XXXX for node ports (needs to be set from the service)

Reply:
fixed